sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -2322,10 +2318,10 @@ def run(self):
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument('--skip_cmis_mgr', action='store_true')
-    parser.add_argument('--enable_sff_mgr', action='store_true')
+    parser.add_argument('--enable_sff_mgr_controlled_tx', action='store_true')
```

Comment:
@longhuan-cisco why we need this new flag? Can you point to the HLD in the PR description?